app/src/main/java/org/oppia/app/HomeFragment.kt

```diff
@@ -29,7 +30,8 @@ class HomeFragment : Fragment() {
       it.viewModel = viewModel
       it.lifecycleOwner = this
     }
-
+    Logger.e("TAG", "Sample log message");
```

Comment:
This & the next line look like test code--please remove.

However, there is a Log statement below that can be replaced with the new logger. Can you update that & migrate all other log statements to Logger?

Reply:
yes